openapi.yaml

```diff
@@ -525,343 +605,340 @@ parameters:
       - RH
       - EF
       - MP
+    description: |
+      Filter by phone type. Common phone types and their descriptions:
+      - `CM` - Current
+      - `PA` - Student Alternate Contact
+      - `RK` - Financial Aid Address
+      - `MP` - Mobile Phone
+responses:
+  '400':
+    description: Bad request
+    schema:
+      $ref: '#/definitions/ErrorResult'
+  '404':
+    description: Resource not found
+    schema:
+      $ref: '#/definitions/ErrorResult'
+  '409':
+    description: Type conflict
+    schema:
+      $ref: '#/definitions/ErrorResult'
+  '500':
+    description: Internal server error
+    schema:
+      $ref: '#/definitions/ErrorResult'
+securityDefinitions:
+  OAuth2:
+    type: oauth2
+    flow: application
+    tokenUrl: https://api.oregonstate.edu/oauth2/token
+    scopes:
+      full: Full access to the API
 definitions:
-  NewPersonResultObject:
+  PersonType:
+    type: string
+    example: person
+  JobType:
+    type: string
+    example: job
+  SsnType:
+    type: string
+    example: ssn
+  MealPlanType:
+    type: string
+    example: meal-plans
+  AddressType:
+    type: string
+    example: addresses
+  PhoneType:
+    type: string
+    example: phones
+  EmailType:
+    type: string
+    example: emails
+  PersonPostAttributes:
+    type: object
+    required:
+    - firstName
+    - lastName
+    - birthDate
+    - sex
     properties:
-      data:
-        $ref: '#/definitions/NewPersonResourceObject'
-  NewPersonResourceObject:
+      firstName:
+        type: string
+        description: First name of person
+        example: John
+      lastName:
+        type: string
+        description: Last name of person
+        example: Smith
+      middleName:
+        type: string
+        description: Middle name of person
+        example: Adam
+      birthDate:
+        type: string
+        format: date
+        description: Birthday of person
+        example: 1970-1-21
+      citizen:
+        type: string
+        enum: [FN, N, R, S, C]
+        description: |
+          Person's citizen code
+          - `FN` - Foreign National
+          - `N` - Non Resident Alien
+          - `R` - Resident Alien
+          - `S` - Substantial Presence Alien
+          - `C` - Citizen
+      sex:
+        type: string
+        description: |
+          Person's sex code
+          - `M` - Male
+          - `F` - Female
+          - `N` - Non-specified
+  PersonAttributes:
```

Comment:
What do you think those fields I left in the comment of JIRA ticket?

* ethnicity
* marital status
* deceased status
* veteran status
* driver license
* etc.